index.html

```diff
@@ -10,188 +10,223 @@
+            <ul class="tec-section">
+                <li class="tec lang">
+                    <h2>Langguages</h2>
+                    <ul class="featureslist"> 
+                    <li>Javascript</li>
+                    <li>Ruby</li>
+                    <li>Html</li>
+                    <li>Css</li>
+                    </ul>
+                </li>
+                <li class="tec farme ">
+                    <h2>Frameworks</h2>
+                    <ul class="featureslist"> 
+                    <li>Bootstrap</li>
+                    <li>Ruby on Rails</li>
+                    <li>RSPec</li>
+                    <li>CapyBara</li>
+                    <li>Selenium</li>
+                    </ul>
+                </li>
+                <li class="tec skills">
+                    <h2>Skills</h2>
+                    <ul class="featureslist"> 
+                    <li>Codekit</li>
+                    <li>Github</li>
+                    <li>Codepen</li>
+                    <li>Gitlab</li>
+                    <li>Terminal</li>
+                    </ul>
+                </li>
+              </ul>
+        </div>
+    </section>
+    
    
-    <hr class="hrbar">
-
-    <div class="socialicons">
-        <ul class="so-icons">
-            <li><a href="#"><i class="fab fa-github"></i></a></li>
-            <li><a href="#"><i class="fab fa-linkedin-in"></i></a></li>
-            <li><a href="#"><i class="fab fa-twitter"></i></a></li>
-            <li><a href="#"><i class="fab fa-facebook"></i></a></li>
-            <li><a href="#"><i class="fab fa-medium-m"></i></a></li>
-        </ul>
+    <div class="work-socio">
+        <div class="work workl">
+            <div class="project-setion">
+                <i class="fa fa-arrow-down work-icon"></i>
+                <h6 class="work-title"><a href="#">Work</a></h6>
+            </div>
+        </div>
+        <div class="socialicons">
+            <ul class="so-icons">
+                <li><a href="#"><i class="fab fa-github"></i></a></li>
+                <li><a href="#"><i class="fab fa-linkedin-in"></i></a></li>
+                <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+                <li><a href="#"><i class="fab fa-facebook"></i></a></li>
+                <li><a href="#"><i class="fab fa-medium-m"></i></a></li>
+            </ul>
+        </div>
+        <div class="work ">
+            <div class="project-setion">
+                <i class="fa fa-arrow-down work-icon"></i>
+                <h6 class="work-title"><a href="#">Work</a></h6>
+            </div>
+        </div>
     </div>
+
+
+    
     <section class="recnt-projects">
-        <div class="project-setion">
-            <i class="fa fa-arrow-down work-icon"></i>
-            <h6 class="work-title"><a href="#">Work</a></h6>
-        </div>
-        <h1 class="recent-project-title">MY RECENT WORK</h1>
+        <h2 class="recent-project-title">MY RECENT WORK</h2>
         <hr class="hrbar">
```

Comment:
- [ ] Let's use the correct styling for this element in the desktop version so that your design matches that of your chosen template on the desktop version. Please fix the style using the typography below which I got from your chosen template on Figma.

<img width="104" alt="Loom_zwwbkhcZho" src="https://user-images.githubusercontent.com/81468675/149388232-241e71c0-466b-4f1c-9aff-37a46294326e.png">

Reply:
Thanks for your comment, fixed typography for project title as per Figma design